packages/web/src/components/static-image/StaticImage.tsx

```diff
@@ -0,0 +1,91 @@
+import { ComponentPropsWithoutRef } from 'react'
+
+import { SquareSizes } from '@audius/common/models'
+import { Box } from '@audius/harmony'
+import {
+  DiscoveryNodeSelector,
+  productionConfig,
+  stagingConfig,
+  developmentConfig,
+  StorageNodeSelector,
+  AppAuth
+} from '@audius/sdk'
+
+import { env } from 'services/env'
+
+export type StaticImageProps = {
+  // Image CID
+  cid?: string | null
+  // Size of image to select
+  size?: SquareSizes
+  // Set height and width of wrapper container to 100%
+  fullWidth?: boolean
+  // Classes to apply to the wrapper
+  wrapperClassName?: string
```

Comment:
total nit, but this could just be "className" the benefit of that is then when you render `<StaticImage />` you can apply a css prop instead of css-module. that gets converted to "className" ultimately, if that makes sense. i get this is mimicking DynamicImage, but so ya know

Reply:
this is the classname for the wrapping div though. any css prop or class name added to the component would be applied to the img properly with this setup

Reply:
ohh gotcha, nice